starlark/int_test.go

```diff
@@ -0,0 +1,54 @@
+// Copyright 2017 The Bazel Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package starlark
+
+import (
+	"math"
+	"testing"
+)
+
+func TestIntOpts(t *testing.T) {
+	f := maybeBig
+	for i, test := range []Int{
```

Comment:
It would be nice to show (and assert) the expected result as a string:

{f(math.MaxInt32).Add(f(1)), "0x100000000"},
etc